R/trust_types.R

```diff
@@ -0,0 +1,37 @@
+# This script joins the trust type (from the CQC rating data) onto the open trust data in the geographr package  
+
+# Load libraries 
+library(geographr)
+library(sf)
+library(tidyverse)
+library(readODS)
+
+source("R/utils.R") # for download_file()
+
+# Downloading CQC rating data as has information on what is the primary type of care trust provides 
+# Landing page of data: https://www.cqc.org.uk/about-us/transparency/using-cqc-data#directory
+tf <- download_file("https://www.cqc.org.uk/sites/default/files/04_January_2022_Latest_ratings.ods", "ods")
```

Comment:
```suggestion
tf <- 
  download_file(
    "https://www.cqc.org.uk/sites/default/files/04_January_2022_Latest_ratings.ods",
    "ods"
  )
```
Where possible, try and limit line lengths to 80 characters to aid legibility.